README.md

```diff
@@ -10,121 +12,42 @@ The serialization and deserialization code that is generated by this project is
 optimized and is between **10 and 25 times faster**<sub>[why?](#why-is-it-faster-than-the-apache-implementation)</sub> than the code generated by the
 Apache Erlang implementation.
 
-## Setup
+## Project Status
 
-Start by adding this package to your project as a dependency:
+[Version 2.0](https://github.com/pinterest/elixir-thrift/milestone/1) is under
+actively development and should be released soon. It is a complete rewrite
+that drops the Apache Thrift dependency and implements everything in pure
+Elixir.
 
-```elixir
-{:thrift, "~> 2.0"}
-```
+## Getting Started
 
-Or to track the GitHub master branch:
+Until version 2.0 is released, you'll need to track the master branch
+directly:
```

Comment:
Should we release a `2.0.0-rc1` so folks can use Hex instead of GitHub directly? I see our mix file has `2.0.0-dev`, but having `rc` releases would give more flexibility for releasing more than one development version.

Reply:
Yes, I think we're getting pretty close to that point. I'd like to get some more of the documentation updated before we invite the first round of feedback.